packages/sdks/overrides/react-native/src/functions/transform-block-properties.ts

```diff
@@ -26,6 +27,14 @@ export function transformBlockProperties({
       if (el && !(el instanceof Text)) {
         el.setAttribute('builder-id', id);
         el.classList.add(id);
+        // Support inline text editing for React Native SDK
+        if (block.component?.name === 'Text') {
+          const textElement = el.querySelector('[dir="auto"]');
```

Comment:
the `dir="auto"` selector doesn't completely fix inline editing. it will only grab the first `div` to match it, which is not enough. See https://www.loom.com/share/ad2bfcbd24594d3283ee5a3fb3a0bc67

What you need to do is set `builder-text` on the root `View`/`div` created by `react-native-render-html`. You can do that by grabbing the firstElementChild of `el`

```suggestion
          const textElement = el.firstElementChild
```

Reply:
For now, I would remove this inline editing code fix completely until we figure out how to properly get it working.

Reply:
thanks @samijaber , have reverted this change for now. Will test this out thoroughly and create a separate PR because if you check my loom (it propagates properly):
https://www.loom.com/share/425267146fb04c24ac7b6c1d636e9739

Reply:
You're right....interesting.